dbus-macros/.gitignore

```diff
@@ -0,0 +1 @@
+wip/**
```

Comment:
Not needed :-)

Reply:
This is for trybuild dependency, it dumps stderr into wip/testname.stderr file if it can't find tests/ui/testname.stderr, so it's useful